core/integration/features/verified/0042-LIQF-fees_rewards.feature

```diff
@@ -289,11 +282,26 @@ Feature: Test liquidity provider reward distribution; Should also cover liquidit
       | party2 | USD   | 100000000  |
 
     And the parties submit the following liquidity provision:
-      | id  | party | market id | commitment amount | fee   | lp type    |
-      | lp1 | lp1   | ETH/MAR22 | 8000              | 0.001 | submission |
-      | lp1 | lp1   | ETH/MAR22 | 8000              | 0.001 | submission |
-      | lp1 | lp1   | ETH/MAR22 | 8000              | 0.001 | submission |
-      | lp1 | lp1   | ETH/MAR22 | 8000              | 0.001 | submission |
+      | id  | party | market id | commitment amount | fee    | lp type    | error                           |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | -0.001 | submission | invalid liquidity provision fee |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | -0.001 | submission | invalid liquidity provision fee |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | -0.001 | submission | invalid liquidity provision fee |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | -0.001 | submission | invalid liquidity provision fee |
+
+    And the parties submit the following liquidity provision:
+      | id  | party | market id | commitment amount | fee | lp type    |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | 0   | submission |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | 0   | submission |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | 0   | submission |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | 0   | submission |
+
+    And the parties submit the following liquidity provision:
+      | id  | party | market id | commitment amount | fee   | lp type   |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | 0.001 | amendment |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | 0.001 | amendment |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | 0.001 | amendment |
+      | lp1 | lp1   | ETH/MAR22 | 8000              | 0.001 | amendment |
```

Comment:
Why do we need four rows for each of these?

Reply:
good point, delete 3 rows :)